### PR TITLE
Update ko_kr.json

### PR DESCRIPTION
**Pull request by Hu-Myo**
I've just confirmed that it's been added
I've finished updating, can you merge it?

### CODE REVIEW

**Comment by MoriyaShiine** [comment]
Looks like there's conflicts, can you see what's up with that?

**Comment by Hu-Myo** [comment]
I rewrote it from scratch using en-us.json
I guess it's because it's different from my fork's en-us.json?

I don't think there's anything else except for the part I changed to Korean, but did I make a mistake?
I don't know how to use github, so I don't know why the conflicts happened.

**Comment by Hu-Myo** [comment]
Oh, there are some big changes.
There were some mistranslations, so I corrected them, saw what was added and deleted in the en-us.json, translated it back from the en-us.json, pasted it in, and there were conflicts.

I added it to "enchancement.midnightconfig" and I thought it would be weird if I translated this part, so I fixed it back to English, but can you check it?
I think you can assume I just updated it.

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 1]
```diff
@@ -128,30 +128,33 @@
...
-  "enchancement.midnightconfig.negateEnderPearlDamage": "엔더 진주 피해 무효화",
-  "enchancement.midnightconfig.safeChanneling": "집전 피해 무효화(집전으로 인한 번개 데미지 무효)",
-  "enchancement.midnightconfig.tridentsReturnFromVoid": "삼지창이 공허로 떨어져도 돌아옴",
-  "enchancement.midnightconfig.weakerFireAspect": "약한 발화",
-  "enchancement.midnightconfig.weakerGoldenApple": "약한 황금 사과",
-  "enchancement.midnightconfig.weakerPotions": "약한 물약",
-  "enchancement.midnightconfig.weaponEnchantmentCooldownRequirement": "무기 마법 부여 쿨타임 요구",
-  "enchancement.midnightconfig.maxExtractingBlocks": "섬세한 채굴에 대한 최대 블록 갯수",
-  "enchancement.midnightconfig.maxLumberjackBlocks": "벌목꾼에 대한 최대 블록 갯수",
-  "enchancement.midnightconfig.maxLumberjackHorizontalLength": "벌목꾼에 대한 최대 가로 길이",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json]: Changed what was written

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 2]
```diff
@@ -1,115 +1,157 @@
 {
-  "entity.enchancement.frozen_player": "플레이어",
+  "entity.enchancement.frozen_player": "얼어붙은 플레이어",
   "entity.enchancement.ice_shard": "얼음 조각",
-  "entity.enchancement.brimstone": "혈사",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:4]: Mistranslations

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 3]
```diff
@@ -1,115 +1,157 @@
...
+  "entity.enchancement.brimstone": "발광석",
   "entity.enchancement.amethyst_shard": "자수정 조각",
-  "entity.enchancement.torch": "횟불",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:6]: Typos

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 4]
```diff
@@ -1,115 +1,157 @@
 {
-  "entity.enchancement.frozen_player": "플레이어",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:2]: Mistranslations

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 5]
```diff
@@ -1,115 +1,157 @@
...
-  "enchantment.minecraft.channeling.desc.redirect_melee": "폭풍우가 치는동안 삼지창이 번개를 소환합니다. 근접 공격시 대상에게 화염 피해를 입힙니다.",
-  "enchantment.minecraft.channeling.desc.redirect_thunderless": "삼지창을 던져 대상에게 맞추면, 번개를 소환합니다.",
-  "enchantment.minecraft.channeling.desc.redirect_melee_thunderless": "삼지창을 던져 대상에게 맞추면, 번개를 소환합니다. 근접 공격시 대상에게 화염 피해를 입힙니다.",
-  "enchantment.minecraft.luck_of_the_sea.desc.redirect": "낚시할때 보물을 얻을 확률이 증가하고 낚싯바늘을 다시 건들일때까지 걸리는 시간을 줄입니다.",
-  "enchantment.minecraft.unbreaking.desc.redirect": "내구도가 줄어들 확률을 줄입니다.",
-  "advancements.adventure.two_birds_one_arrow.description.redirect": "쇠뇌를 이용하여 한방에 두마리의 팬텀을 처치하세요",
-  "enchantment.enchancement.assimilation": "비상식량",
-  "enchantment.enchancement.assimilation.desc": "음식이 채워주는 배고픔 수치만큼 배 고플때 음식을 자동으로 먹습니다. (Off Hand) 슬롯에 음식을 들고 있을 경우 이 음식만 먹게 됩니다. 음식과 음료의 소비 속도가 50% 빨라집니다."
-  "enchantment.enchancement.perception": "통찰",
-  "enchantment.enchancement.perception.desc": "어둠 속에서의 가시성을 높힙니다. 벽 너머에 있는 16블록 이내의 대상을 볼 수 있습니다.",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:18]: Changed what was written

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 6]
```diff
@@ -1,115 +1,157 @@
...
-  "enchantment.enchancement.strafe": "순보",
-  "enchantment.enchancement.strafe.desc": "달리기 키를 두번 누르면 이동 방향으로 밀립니다. 또한 공중 이동성을 증가시킵니다.",
-  "enchantment.enchancement.wardenspine": "고슴도치",
-  "enchantment.enchancement.wardenspine.desc": "뒤에서 들어오는 데미지를 80% 감소시키고, 공격자에게 데미지를 반사합니다.",
-  "enchantment.enchancement.dash": "대시",
-  "enchantment.enchancement.dash.desc": "공중에서 웅크리기를 이용하여 바라보는 방향으로 즉시 밀려납니다. 대시 직후 점프하면 조금 더 멀리 이동 할 수 있습니다.",
-  "enchantment.enchancement.slide": "슬라이딩",
-  "enchantment.enchancement.slide.desc": "질주 대신 계속 슬라이딩하며 미끄러집니다. 슬라이딩 도중에 점프하면 앞으로 조금 더 도약합니다. 공중에서 웅크리기를 사용하면 지면에 강하게 돌진하며 대상를 밀어내고 낙하 피해를 무시합니다.",
-  "enchantment.enchancement.buoy": "수중부양",
-  "enchantment.enchancement.buoy.desc": "물 위를 걸어다닐 수 있고, 물 아래에서 걸어다닐 수 있습니다. 물 속에서 점프하면 위로 빠르게 솟구칩니다. 하지만 수영할 수 없습니다.",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:32]: Changed what was written

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 7]
```diff
@@ -1,115 +1,157 @@
...
+  "enchantment.enchancement.wardenspine": "수호자의 등",
+  "enchantment.enchancement.wardenspine.desc": "공격자에게 피해를 입히고 뒤에서 받는 피해를 크게 감소합니다.",
+  "enchantment.enchancement.dash": "대쉬",
+  "enchantment.enchancement.dash.desc": "공중에서 웅크리기를 이용하여 바라보는 방향으로 자신을 날려보냅니다.",
+  "enchantment.enchancement.slide": "슬라이드",
+  "enchantment.enchancement.slide.desc": "뛰는 것 대신 달리기 키를 누르고 있으면 계속 미끄러집니다. 공중에서 웅크리면 지면에 강하게 돌진하며 대상을 밀어내고 낙하 피해를 무시합니다.",
+  "enchantment.enchancement.buoy": "부표",
+  "enchantment.enchancement.buoy.desc": "물 속을 걸어다니며, 웅크리면 수면 아래로 내려갑니다. 물에 잠긴 상태로 점프 키를 누르면 수면 위로 빠르게 떠오릅니다. 헤엄칠 수 없습니다.",
   "enchantment.enchancement.bouncy": "통통이",
-  "enchantment.enchancement.bouncy.desc": "웅크리지 않고 착지하면 착지하던 속도에 비례하는 속도로 다시 튀어오릅니다. 웅크리고 있다면 점프 부스트를 충전 할 수 있습니다. 또한 낙하 데미지를 무시합니다.",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:34]: Changed what was written

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 8]
```diff
@@ -1,115 +1,157 @@
...
-  "enchantment.enchancement.gale": "3단 점프",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:35]: Mistranslations

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 9]
```diff
@@ -1,115 +1,157 @@
...
+  "enchantment.enchancement.bouncy.desc": "낙하 피해를 무시합니다. 웅크리지 않은 상태에서 지면에 닿으면 다시 튀어오릅니다. 지면에 닿은 상태에서 웅크리면 슈퍼 점프를 충전하기 시작합니다.",
+  "enchantment.enchancement.gale": "강풍",
   "enchantment.enchancement.gale.desc": "공중에서 2번 더 점프할 수 있습니다.",
   "enchantment.enchancement.berserk": "광폭화",
-  "enchantment.enchancement.berserk.desc": "빈 하트(체력)만큼 0.5데미지 배율로 최대 50% 더 많은 데미지를 줍니다.",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:38]: Changed what was written

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 10]
```diff
@@ -1,115 +1,157 @@
...
+  "enchantment.enchancement.berserk.desc": "체력 손실에 비례하여 공격력이 증가합니다.",
   "enchantment.enchancement.frostbite": "동상",
-  "enchantment.enchancement.frostbite.desc": "대상를 타격하면 이동 속도를 감소시키고 죽이면 얼립니다. 얼어붙은 대상은 움직일 수 있고 산산조각 나며, 산산조각 날때 얼음 조각이 생성되고 이에 맞은 대상은 데미지를 입거나 얼어버립니다.",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:40]: Changed what was written

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 11]
```diff
@@ -1,115 +1,157 @@
...
+  "enchantment.enchancement.frostbite.desc": "공격한 몹을 얼립니다. 처치하면 움직이거나 부술 수 있는 얼음 덩어리로 변하며 부숴지면 얼음 조각을 날려 맞은 대상을 얼려버립니다.",
   "enchantment.enchancement.chaos": "혼돈",
-  "enchantment.enchancement.chaos.desc": "쏠 때 일반 화살에 무작위로 부정적인 효과가 부여됩니다, 웅크리기를 사용하면 긍정적힌 효과가 부여됩니다.",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:42]: Changed what was written

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 12]
```diff
@@ -1,115 +1,157 @@
...
   "enchantment.enchancement.delay": "딜레이",
-  "enchantment.enchancement.delay.desc": "화살은 15초가 지나거나, 때리기 전까지 발사되지 않습니다. 5초동안 발사 되지 않으면 최대 2.5배까지 데미지가 추가된다.",
-  "enchantment.enchancement.phasing": "무중력",
-  "enchantment.enchancement.phasing.desc": "화살은 중력의 영향을 받지 않습니다. 3개의 블록을 통과하면 화살은 중력을 회복합니다.",
-  "enchantment.enchancement.brimstone": "혈사포",
-  "enchantment.enchancement.brimstone.desc": "방어력을 무시하는 날카로운 레이저를 발사합니다. 쇠뇌 장전 속도가 느려지고 대상이 멀수록 데미지가 늘어납니다. 화살 대신 하트(체력)를 소모합니다.",
-  "enchantment.enchancement.scatter": "샷건",
-  "enchantment.enchancement.scatter.desc": "자수정 조각을 (Off Hand) 슬롯에 들고 쇠뇌를 장전할 수 있습니다. 자수정 조각은 발사되면 터지며 전방에 넓은 범위로 데미지를 줍니다.",
-  "enchantment.enchancement.torch": "불 공격",
-  "enchantment.enchancement.torch.desc": "횟불을 (Off Hand) 슬롯에 들고 빠르게 쇠뇌에 장전할 수 있습니다. 발사된 횟불에 맞은 대상은 2초간 화염에 휩사이고, 웅크리면서 발사하면 바닥에 횟불이 설치됩니다.",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:52]: Changed what was written and Mistranslations

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 13]
```diff
@@ -1,115 +1,157 @@
...
+  "enchantment.enchancement.phasing.desc": "화살은 블록을 통과하기 전까지 중력의 영향을 받지 않습니다.",
+  "enchantment.enchancement.brimstone": "발광석",
+  "enchantment.enchancement.brimstone.desc": "방어구를 무시하고 충전량과 거리에 따라 공격력이 증가하는 관통 레이저를 발사합니다. 발사하면 체력이 소모됩니다.",
+  "enchantment.enchancement.scatter": "흩뿌리기",
+  "enchantment.enchancement.scatter.desc": "자수정 조각을 장전하여 발사 시 사방으로 흩뿌려집니다.",
+  "enchantment.enchancement.torch": "횃불",
+  "enchantment.enchancement.torch.desc": "횃불을 빠르게 장전하고 발사하면 대상을 화염에 휩싸이게 만듭니다. 웅크린 상태로 발사하면 떨어진 블록 위에 횃불이 설치됩니다.",
   "enchantment.enchancement.leech": "거머리",
-  "enchantment.enchancement.leech.desc": "근접 공격시 절반의 하트(체력)을 회복합니다. 원거리 공격시 6초동안 맞은 대상에게 달라붙어 피해를 입히고, 초당 하트(체력)의 절반을 회복합니다.",
-  "enchantment.enchancement.warp": "축지법",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:55]: Changed what was written and Mistranslations

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 14]
```diff
@@ -1,115 +1,157 @@
...
+  "enchantment.enchancement.leech.desc": "피해를 입히면 생명력이 회복됩니다. 던진 삼지창이 일시적으로 대상에게 달라붙어 피해를 입힙니다.",
+  "enchantment.enchancement.warp": "워프",
   "enchantment.enchancement.warp.desc": "삼지창에 맞은 대상이 없는 경우, 삼지창의 위치로 순간이동 합니다.",
   "enchantment.enchancement.extracting": "섬세한 채굴",
   "enchantment.enchancement.extracting.desc": "광석에서 광물만 빼내고 돌로 변환합니다. 웅크리는 동안에는 사용할 수 없습니다.",
   "enchantment.enchancement.molten": "제련",
   "enchantment.enchancement.molten.desc": "캔 모든 블록을 제련한 상태로 캡니다. 웅크리는 동안에는 사용할 수 없습니다.",
   "enchantment.enchancement.beheading": "참수",
-  "enchantment.enchancement.beheading.desc": "대상이 죽으면 머리를 떨어트릴 확률을 증가시킵니다.",
-  "enchantment.enchancement.lumberjack": "벌목",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:63]: Changed what was written and Mistranslations

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 15]
```diff
@@ -1,115 +1,157 @@
...
+  "enchantment.enchancement.beheading.desc": "몹과 플레이어는 일정 확률로 머리를 떨어트릴 수 있습니다.",
+  "enchantment.enchancement.lumberjack": "벌목꾼",
   "enchantment.enchancement.lumberjack.desc": "나무 하나를 캐면 연결되어 있는 모든 나무 블록이 아이템화 됩니다. 웅크리는 동안에는 사용할 수 없습니다.",
   "enchantment.enchancement.bury": "매장",
-  "enchantment.enchancement.bury.desc": "바닥이 충분히 부드럽다면 상호 작용한 대상을 바닥에 묻습니다.",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:66]: Changed what was written

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 16]
```diff
@@ -1,115 +1,157 @@
...
+  "enchantment.enchancement.bury.desc": "바닥이 충분히 부드럽다면 대상을 바닥에 묻습니다.",
   "enchantment.enchancement.scooping": "적출",
-  "enchantment.enchancement.scooping.desc": "삽은 1의 추가 피해를 입힐 수 있으며, 몹을 죽이면 더 많은 전리품을 떨어트립니다.",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:68]: Changed what was written

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 17]
```diff
@@ -1,115 +1,157 @@
...
+  "enchantment.enchancement.scooping.desc": "추가 피해를 입히고 몹의 떨어트리는 아이템 양이 증가합니다.",
   "enchantment.enchancement.disarm": "무장해제",
   "enchantment.enchancement.disarm.desc": "대상이 가진 아이템을 훔치거나 비활성화 합니다.",
-  "enchantment.enchancement.grapple": "그래플",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:71]: Mistranslations

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 18]
```diff
@@ -1,115 +1,157 @@
...
-  "subtitles.enchancement.entity.shard.shatter": "Shard shatters",
-  "subtitles.enchancement.entity.generic.strafe": "Entity strafes",
-  "subtitles.enchancement.entity.generic.dash": "Entity dashes",
-  "subtitles.enchancement.entity.generic.impact": "Entity impacts",
-  "subtitles.enchancement.entity.generic.air_jump": "Entity air jumps",
-  "subtitles.enchancement.entity.generic.freeze": "Entity freezes",
-  "subtitles.enchancement.entity.generic.teleport": "Entity teleports",
-  "subtitles.enchancement.entity.generic.bury": "Entity buried",
-  "subtitles.enchancement.block.ore.extract": "Ores extracted",
-  "subtitles.enchancement.block.generic.smelt": "Block smelts",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:84]: Mistranslations

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 19]
```diff
@@ -1,115 +1,157 @@
...
+  "subtitles.enchancement.block.ore.extract": "광물이 추출됨",
+  "subtitles.enchancement.block.generic.smelt": "블록이 제련됨",
+  "death.attack.enchancement.life_drain": "%1$의 생명의 불꽃이 꺼졌습니다",
+  "death.attack.enchancement.life_drain.item": "%1$은(는) %2$와(과) %3$을 휘두르며 싸우는 동안 생명이 사그라들었습니다",
+  "death.attack.enchancement.life_drain.player": "%1$이(가) %2$과(와) 싸우다가 결국 생명을 빼앗겼습니다",
+  "key.categories.enchancement": "Enchancement",
+  "key.enchancement.strafe": "회피 기동",
   "tooltip.enchancement.experience_level_cost": "필요한 경험치 비용: %d 레벨",
   "tooltip.enchancement.lapis_lazuli_cost": "필요한 청금석 갯수: %d 청금석",
-  "tooltip.enchancement.cannot_assimilate": "음식을 먹을 수 없습니다",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:87]: Typos

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 20]
```diff
@@ -1,115 +1,157 @@
...
+  "tooltip.enchancement.cannot_assimilate": "적응에 의해 음식을 먹을 수 없습니다",
   "enchancement.midnightconfig.title": "Enchancement",
-  "enchancement.midnightconfig.allowedEnchantments": "허용된 Enchancement",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:89]: Changed what was written

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 21]
```diff
@@ -1,115 +1,157 @@
...
+  "enchancement.midnightconfig.category.client": "Client",
+  "enchancement.midnightconfig.disallowedEnchantments": "허용된 Enchantments",
   "enchancement.midnightconfig.invertedList": "반전된 리스트",
-  "enchancement.midnightconfig.overhaulEnchantingTable": "마법 부여대 오버홀",
-  "enchancement.midnightconfig.allowTreasureEnchantmentsInEnchantingTable": "마법 부여대에서 보물 인챈트 허용",
-  "enchancement.midnightconfig.experienceLevelCost": "필요한 경험치 레벨 비용",
-  "enchancement.midnightconfig.lapisLazuliCost": "필요한 청금석 갯수",
-  "enchancement.midnightconfig.singleEnchantmentMode": "단일 인챈트 모드",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:95]: Mistranslations

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 22]
```diff
@@ -1,115 +1,157 @@
...
+  "enchancement.midnightconfig.overhaulEnchantingTable": "정밀한 마법 부여대",
+  "enchancement.midnightconfig.allowTreasureEnchantmentsInEnchantingTable": "마법 부여대에서 보물 마법 허용",
+  "enchancement.midnightconfig.enchantmentLimit": "마법 부여 한도",
   "enchancement.midnightconfig.singleLevelMode": "단일 레벨 모드",
   "enchancement.midnightconfig.allowInfinityOnCrossbows": "쇠뇌에 무한 인챈트 허용",
   "enchancement.midnightconfig.allTridentsHaveLoyalty": "삼지창에 충성 자동 적용(삼지창 자동 회수)",
   "enchancement.midnightconfig.channelingIgnitesOnMelee": "집전을 근거리에서 점화(번개 근거리 소환)",
   "enchancement.midnightconfig.channelingWorksWhenNotThundering": "집전 발동시 천둥 제거(번개 제거 효과는 그대로 적용)",
   "enchancement.midnightconfig.fireAspectWorksAsFlintAndSteel": "발화를 부싯돌과 부시처럼 사용",
-  "enchancement.midnightconfig.freeEnchantedBookMerging": "마법이 부여된 책 비용 무료화",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:102]: Typos

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 23]
```diff
@@ -1,115 +1,157 @@
...
+  "enchancement.midnightconfig.freeEnchantedBookMerging": "마법이 부여된 책 비용 무료",
   "enchancement.midnightconfig.luckOfTheSeaHasLure": "바다의 행운에 미끼 포함",
   "enchancement.midnightconfig.negateEnderPearlDamage": "엔더 진주 피해 무효화",
   "enchancement.midnightconfig.safeChanneling": "집전 피해 무효화(집전으로 인한 번개 데미지 무효)",
   "enchancement.midnightconfig.tridentsReturnFromVoid": "삼지창이 공허로 떨어져도 돌아옴",
   "enchancement.midnightconfig.weakerFireAspect": "약한 발화",
   "enchancement.midnightconfig.weakerGoldenApple": "약한 황금 사과",
-  "enchancement.midnightconfig.weakerPotions": "약한 포션",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:109]: Mistranslations

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 24]
```diff
@@ -1,115 +1,157 @@
...
+  "enchancement.midnightconfig.weakerPotions": "약한 물약",
   "enchancement.midnightconfig.weaponEnchantmentCooldownRequirement": "무기 마법 부여 쿨타임 요구",
   "enchancement.midnightconfig.maxExtractingBlocks": "섬세한 채굴에 대한 최대 블록 갯수",
-  "enchancement.midnightconfig.maxLumberjackBlocks": "벌목에 대한 최대 블록 갯수",
-  "enchancement.midnightconfig.maxLumberjackHorizontalLength": "벌목에 대한 최대 가로 길이",
-  "enchancement.midnightconfig.unbreakingChangesFlag": "내구성 변경 Flag"
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json:114]: Mistranslations

File: src/main/resources/assets/enchancement/lang/ko_kr.json [hunk 25]
```diff
@@ -128,30 +128,33 @@
...
+  "enchancement.midnightconfig.tridentsReturnFromVoid": "Tridents Return From Void",
+  "enchancement.midnightconfig.weakerFireAspect": "Weaker Fire Aspect",
+  "enchancement.midnightconfig.weakerGoldenApple": "Weaker Golden Apple",
+  "enchancement.midnightconfig.weakerPotions": "Weaker Potions",
+  "enchancement.midnightconfig.weaponEnchantmentCooldownRequirement": "Weapon Enchantment Cooldown Requirement",
+  "enchancement.midnightconfig.maxExtractingBlocks": "Max Extracting Blocks",
+  "enchancement.midnightconfig.maxLumberjackBlocks": "Max Lumberjack Blocks",
+  "enchancement.midnightconfig.maxLumberjackHorizontalLength": "Max Lumberjack Horizontal Length",
   "enchancement.midnightconfig.unbreakingChangesFlag": "Unbreaking Changes Flag",
-  "enchancement.midnightconfig.enchantmentDescriptions": "Enchantment 설명",
```
Comment (Hu-Myo) [inline src/main/resources/assets/enchancement/lang/ko_kr.json]: Recover translations for accurate communication

**Comment by MoriyaShiine** [comment]
There have been a lot of changes since this PR was made, can you please update all of the entries

**Comment by Hu-Myo** [comment]
I've just updated, can you confirm?

**Comment by MoriyaShiine** [comment]
Seems to be everything, thanks!

**Pull request merged**